api/handlers/metadata.go

```diff
@@ -93,19 +94,23 @@ func (h *metadataHandler) getMetadata() middleware.Responder {
 func (h *metadataHandler) getBridgeVersion(namespace string) (string, error) {
 
 	if h.k8sClient == nil {
-		return "", fmt.Errorf("unable to get bridge version")
+		return "", fmt.Errorf("no K8s client available")
 	}
 	bridgeDeployment, err := h.k8sClient.AppsV1().Deployments(namespace).Get(context.TODO(), "bridge", metav1.GetOptions{})
 	if err != nil {
-		return "", fmt.Errorf("unable to get bridge version %w", err)
+		return "", fmt.Errorf("unable to get bridge version: %w", err)
+	}
+
+	if len(bridgeDeployment.Spec.Template.Spec.Containers) == 0 {
+		return "", errors.New("bridge deployment does not contain any image")
```

Comment:
```suggestion
		return "", fmt.Errorf("bridge deployment does not contain any image")
```
let's try to be consistent

Reply:
if we don't pass any parameters it does not make sense to use `fmt.Errorf`